app/inc/search_options.php

```diff
@@ -12,34 +12,24 @@
 // Checkboxes states
 $check = [];
 
+// Search forms
+$form_checkboxes = array_diff($search->getFormSearchOptions(), ['repo', 'search_type']);
```

Comment:
Since the other options are defined in the class, it might make more sense to move this too inside the class.